### ISSUE TITLE
Tried almost everything. Still wont work.

### ISSUE THREAD

**Issue by Esheii** [Jire/Charlatano#530]
Tried installing 5 different versions of java and charletano itself and everything else that were listed in previous threads here. I completed the build, but when I go ingame and run it, I get the an error in the CMD saying: 

"WARN: [kts] kotlin.script.experimental.dependencies.DependenciesResolver.Nodependencies must have a constructor without required parameters."

Any help?

**Comment by dStroke** [comment]
ignore that, it works 
i have the same it works

**Comment by tamal777** [comment]
reinstall java and recompile it.

**Comment by Esheii** [comment]
@tamal777 
So we're on the same foot, by recompiling it you just mean running the build script right?
Done that numerous times.

@dStroke 
I still can't get it to work ingame. Even checked if the settings were on aswell.

**Comment by rootm0s** [comment]
@Esheii Are you using Windows 10? Make sure Charlatano and CSGO is not running as admin. Worked for me!

**Comment by Esheii** [comment]
@rootm0s Yeah I already tried running both as admin and running both not as admin, still doesn't work for me sadly.

**Comment by Jire** [comment]
Your Steam and CSGO is what you need to .ake sure isn't ran as admin

**Comment by tamal777** [comment]
Charlatano is working fine. I'm using win 7 64bit. Use old version of jdk 8 update 131. It might help.

**Comment by Esheii** [comment]
@tamal777 I am using the the exact version you wrote down since it was in the tutorial video. 

@Jire I know. Obviously tried that. I even tried running steam offline non admin and as admin aswell.

**Comment by MuhammadSajid1** [comment]
If ESP is working the aimbot is probably working too. I don't know why your aimbot etc is not working.

**Comment by MuhammadSajid1** [comment]
Should work too**

**Comment by Esheii** [comment]
@MuhammadSajid1 yeah that is why I find it weird. The glow esp standard setting does not work, but when I enable box esp and skeleton esp they work, just not the glow. I tried disabling all aimbot settings/bunnyhop/noflash etc and nothing works except for box and skeleton esp.

**Comment by Jire** [comment]
@Esheii Sounds very strange to me, usually it's the opposite way, i.e. glow works but box/skeleton do not.

**Comment by Esheii** [comment]
@Jire Ouch that is even weirder. Don't know what to try anymore though, so I'll look this up once in a while to see if theres a solution in other issue posts.
